src/main/pluginsServer.ts

```diff
@@ -176,17 +176,21 @@ export async function startPluginsServer(
                         instanceId: server.instanceId.toString(),
                         lastActivity: server.lastActivity ? new Date(server.lastActivity).toISOString() : null,
                         lastActivityType: server.lastActivityType,
+                        piscina: piscina ? JSON.stringify(getPiscinaStats(piscina)) : null,
                     }
                     Sentry.captureMessage(
-                        `Plugin Server has not ingested events for over ${serverConfig.STALENESS_RESTART_SECONDS} seconds!`,
+                        `Plugin Server has not ingested events for over ${serverConfig.STALENESS_RESTART_SECONDS} seconds! Rebooting.`,
                         {
                             extra,
                         }
                     )
                     console.log(
-                        `Plugin Server has not ingested events for over ${serverConfig.STALENESS_RESTART_SECONDS} seconds!`,
+                        `Plugin Server has not ingested events for over ${serverConfig.STALENESS_RESTART_SECONDS} seconds! Rebooting.`,
                         extra
                     )
+                    setTimeout(() => process.kill(process.pid, 'SIGINT'), 1000)
+                    setTimeout(() => process.kill(process.pid, 'SIGTERM'), 60000)
+                    setTimeout(() => process.kill(process.pid, 'SIGKILL'), 120000)
```

Comment:
We should log this shutdown to StatsD. BTW isn't SIGINT the same as SIGTERM here?

Reply:
Sure, changed to:

```js
                    server.statsd?.increment(`alerts.stale_plugin_server_restarted`)
                    setTimeout(() => process.kill(process.pid, 'SIGTERM'), 1000)
                    setTimeout(() => process.kill(process.pid, 'SIGTERM'), 60000)
                    setTimeout(() => process.kill(process.pid, 'SIGKILL'), 120000)
```

No need to mess around, we're sending a clear signal that it's time to terminate 🕶️